tests/integration/kubernetes/confidential_kbs.sh

```diff
@@ -326,15 +328,14 @@ EOF
 #
 kbs_k8s_svc_host() {
 	if kubectl get ingress -n "$KBS_NS" 2>/dev/null | grep -q kbs; then
-		kubectl get ingress kbs -n "$KBS_NS" \
+		kubectl get ingress "$KBS_SVC_NAME" -n "$KBS_NS" \
```

Comment:
I know that the ingress name is `kbs`, same as  "$KBS_SVC_NAME". But I´d like to have a new variable, `KBS_INGRESS_NAME`.

Just a nit-pic :)

Reply:
@wainersm changes applied